test/configure-server.js

```diff
@@ -48,6 +48,18 @@ exports.init = function(done){
         done(null,{d0:args.d0+'f0'})
       })
 
+      this.add('role:api,cmd:x1',function(args,done){
+        done(null,{x: args.x})
```

Comment:
use a different name other than `done` for the callback, its harder to read with the test `done` function